.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Customized from the template
+(https://github.com/thealphadollar/vanilla-i18n/blob/master/.github/PULL_REQUEST_TEMPLATE.md)
```

Comment:
I think we should keep the original source of the template here. What do you say?

Reply:
Nice @thealphadollar , but this path already redirect to correct source ? whats's is wrong ? can you write the correct source to update ?

Reply:
Source should be original, from where you've taken the template. For instance, in the issue template the link was of golang repository which was correct. We are using the template and not the creators, so we should keep the link to the original source (the creators of the template) to give them due credits.

Reply:
Nice @thealphadollar !! now I understand . I updated !